### ISSUE TITLE
writeback mode stable or not

### ISSUE THREAD

**Issue by samuelxhu** [Frontier314/EnhanceIO#3]
I have read somewhere on the internet (don't remember exactly where) claiming that enhanceIO may loss data under some circumstances. Hence my question is

1) Does someone here ever use enhanceio writeback cache mode in production, and how about reliability?
2) For how long?

I am planing to use enhanceio as the cache solution below Gluster bricks. Just curious about whether i can use enhanceio write back mode.

**Comment by Frontier314** [comment]
I've used writeback on one of my non-root volumes, back when I was maintaining this fork (is no longer maintained). On non-root volumes I did not experience data loss for almost 6 months (then switched fully to SSDs) but on root volumes writeback will indeed lose data, mostly after a couple of days of use.

To conclude, do not use write-back on root volume, on non-root volumes it worked fine.

**Comment by samuelxhu** [comment]
@Frontier314 Thanks a lot for the information. Did you observe write acceleration by using writeback on non-root volumes? noticeable or dramatic?

**Comment by Frontier314** [comment]
Yes, there was an improvement when the cached disk (i.e the mechanical one) had 16MB of cache memory (or less). But as soon as I upgraded the disk to the newer SATA3 with 64MB cache, the improvement was not noticeable anymore, so I disabled it as a precaution measure.
